src/components/application_manager/src/message_helper.cc

```diff
@@ -1344,6 +1344,13 @@ smart_objects::SmartObjectList MessageHelper::CreateAddSubMenuRequestToHMI(
     msg_params[strings::menu_id] = i->first;
     msg_params[strings::menu_params][strings::position] =
         (*i->second)[strings::position];
+
+    if (((*i->second).keyExists(strings::sub_menu_icon)) &&
```

Comment:
@VVeremjova Can we create an alias for (*i->second) at the beginning of loop and use it instead?